cmd/ical-relay/templates/calendar.html

```diff
@@ -111,12 +111,12 @@
                 calendarEnd = date.add(4, "week").day(6);
             }else{
                 document.getElementById("current-date").innerHTML = "KW"+currentDate.format(" w MMMM");
-
                 calendarStart = date.startOf("week").subtract(1, "day");
+                if(date.day() === 0){
+                    calendarStart = calendarStart.add(1,"week")
+                }
                 calendarEnd = calendarStart.add(1, "week");
             }
-            console.log(calendarStart);
```

Comment:
These logs are somewhat useful for debugging and checking what was interpreted from the current date input (location hash). I would keep them.
They could be made more useful by converting them to output something like "Showing events from cStart to cEnd", but that's not necessary.

Reply:
chef hat gesagt soll weg

Reply:
I think @nilsfast was talking about the 
```
console.log(date);
console.log(date.day());
```
you added, not these two ones which were already there. 
@nilsfast please clarify

Reply:
chef = @nilsfast

Reply:
undeclared use of `chef` use `:=` to declare

Reply:
@robske110 correct. Keep the ln. 118 @Anasailon, I think logging the current date in prod is a bit useless

Reply:
and also ln. 119 :)